### DISCUSSION TITLE
Support for TTreeView

### DISCUSSION THREAD

**Discussion by aaovalle** [Embarcadero/DelphiVCL4Python#26] [Q&A]
I tried to export a form from Delphi that included a tree view. When trying to run it from python it threw an error saying "Class TTreeView not found". did I do something wrong? Thanks for your help!

**Comment by lmbelo** [comment]
Hi @aaovalle,

The TTreeView component is not available as part of the wrapped components collection yet.
I will include this wrapper in our roadmap. I hope to get it available in the next release.

**Reply by edgebyte-ai** [reply]
Also need TProgressBar etc..
How to wrap a customized component for python to use?  Any example?

**Reply by danieleteti** [reply]
It would be nice a tutorial/demo about this. Perhaps using a community well known real-world component.

**Reply by aaovalle** [reply]
Awesome! Thanks @lmbelo
When is the next release scheduled?

**Reply by Priyatham10** [reply]
We're planning for a first major stable release with Python second layer on top of binary extension module. The goal is to provide some utility functions to create applications with more intuitiveness and ease. Also, to reach broader Python community.

**Reply by lmbelo** [reply]
@aaovalle you can follow any of the units within the vcl folder with a "Wrap" prefix.
It would be great to get a pull request 🚀

**Reply by lmbelo** [reply]
@danieleteti thank you for your suggestion. I can work in a small tutorial containing the basics for wrapping up components.

**Reply by lmbelo** [reply]
@zwm88 the TProgressBar component is currently available in the DelphiFMX4Python library.
I will add it in our roadmap to be available in a future release.
I hope it to come out quicly :)

**Reply by edgebyte-ai** [reply]
@lmbelo Please add TStatusBar as well.  Thanks!

**Reply by lmbelo** [reply]
@zwm88 I will 💪

**Reply by edgebyte-ai** [reply]
The DelphiFMX was updated one week ago, when will be the new DelphiVCL available?

**Reply by lmbelo** [reply]
Hi @zwm88,

We still didn't define a release date, but we have many of the new features and bug fixes done.

**Reply by edgebyte-ai** [reply]
One month passed again.

**Reply by lmbelo** [reply]
Hi @zwm88,
we have put a lot of effort in many features still under development. There's a lot to do yet, but I'm looking forward to release the TTreeView component and all the others we have mentioned.
